apps/system/js/task_manager.js

```diff
@@ -149,6 +149,7 @@
 
     this.publish('cardviewbeforeshow');
 
+    screen.mozLockOrientation(OrientationManager.defaultOrientation);
```

Comment:
non blocking but `defaultOrientation` should probably be exposed through `Service.registerState`